### PR TITLE
Add Arion - Air Superiority

### PR DESCRIPTION
**Pull request by Spectre907YT**

### CODE REVIEW

File: manifests/Arion-Air Supperiority.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Arion - Air Superiority
+authors: Spectre907 (With help from 1010todd, yuyuko_saigyouji. and gefulltetaubenbrust)
+homepage: https://github.com/Spectre907YT/Arion-Air_Superiority
+license: CC-BY-SA-4.0
```
Comment (MCOfficer) [inline manifests/Arion-Air Supperiority.yaml:4]: Oh, I didn't realize this earlier: CC-BY-SA-4.0 is one-way compatible with GPL3+; so you can have CC-BY-SA-4.0 assets in a GPL3+ plugin, but not the other way round. Part of your plugin [is GPL3+](https://github.com/Spectre907YT/Arion-Air_Superiority/blob/873c60a668feb0cd102a38c9771986b1e6cb0816/copyright.txt#L6), so you can't use CC-BY-SA-4.0 here.

You'll probably want to change this line to
```suggestion
license: GPL-3.0-or-later
```

File: manifests/Arion-Air Supperiority.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+-Xenonauts fighters
+-A bunch of new H2H weapons based on real life weapons
+-Advanced outfits
+-Specialized weapons
+-Superweapons
+-Custom capital ships
+-1 (or 2) new 'short campaigns' and some new jobs
+-ETC
+url: https://github.com/Spectre907YT/Arion-Air_Superiority/archive/refs/tags/3.0.0.zip
+iconUrl: https://github.com/Spectre907YT/Arion-Air_Superiority/blob/main/icon.png
```
Comment (MCOfficer) [inline manifests/Arion-Air Supperiority.yaml:20]: Should be versioned, and a raw url:
```suggestion
iconUrl: https://github.com/Spectre907YT/Arion-Air_Superiority/raw/3.0.0/icon.png
```

**Review by MCOfficer — CHANGES_REQUESTED** [review]

File: manifests/Arion-Air Supperiority.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+-Real life bombers
+-Ace combat super planes
+-Xenonauts fighters
+-A bunch of new H2H weapons based on real life weapons
+-Advanced outfits
+-Specialized weapons
+-Superweapons
+-Custom capital ships
+-1 (or 2) new 'short campaigns' and some new jobs
+-ETC
```
Comment (MCOfficer) [inline manifests/Arion-Air Supperiority.yaml:18]:
```suggestion
description: |
	An Endless Sky plugin that adds a variety of things to endless sky, including:
	-Real life fighters
	-Real life bombers
	-Ace combat super planes
	-Xenonauts fighters
	-A bunch of new H2H weapons based on real life weapons
	-Advanced outfits
	-Specialized weapons
	-Superweapons
	-Custom capital ships
	-1 (or 2) new 'short campaigns' and some new jobs
	-ETC
```

File: manifests/Arion-Air Superiority.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+name: Arion - Air Superiority
+authors: Spectre907 (With help from 1010todd, yuyuko_saigyouji. and gefulltetaubenbrust)
```
Comment (MCOfficer) [inline manifests/Arion-Air Superiority.yaml:2]:
```suggestion
authors: Spectre907 (With help from 1010todd, yuyuko_saigyouji, and gefulltetaubenbrust)
```

Comment (Spectre907YT) [reply manifests/Arion-Air Superiority.yaml:2]: That one is on purpose, that is the persons discord username

**Review by MCOfficer — APPROVED** [review]

**Pull request merged**